<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>clientInfo</class>
 <widget class="QMainWindow" name="clientInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
<<<<<<< HEAD
    <width>333</width>
    <height>347</height>
=======
    <width>334</width>
    <height>436</height>
>>>>>>> 0bcd9ef0c187252ff7022416289f373dd0043275
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #ffffff;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="connectWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>331</width>
<<<<<<< HEAD
      <height>301</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 5px;
background-color: #F2F6FC;</string>
=======
      <height>391</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #F2F6FC;</string>
>>>>>>> 0bcd9ef0c187252ff7022416289f373dd0043275
    </property>
    <widget class="QLineEdit" name="IDinput">
     <property name="geometry">
      <rect>
       <x>40</x>
<<<<<<< HEAD
       <y>60</y>
=======
       <y>100</y>
>>>>>>> 0bcd9ef0c187252ff7022416289f373dd0043275
       <width>251</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #ffffff;
border: 1px #000000;
border-radius: 5px;
<<<<<<< HEAD
padding: 10px;</string>
=======
padding-left: 10px;</string>
     </property>
     <property name="readOnly">
      <bool>false</bool>
>>>>>>> 0bcd9ef0c187252ff7022416289f373dd0043275
     </property>
    </widget>
    <widget class="QLabel" name="IDlabel">
     <property name="geometry">
      <rect>
       <x>40</x>
<<<<<<< HEAD
       <y>30</y>
=======
       <y>60</y>
>>>>>>> 0bcd9ef0c187252ff7022416289f373dd0043275
       <width>251</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: italic 600 10pt &quot;UTM Avo&quot;;
border: none;</string>
     </property>
     <property name="text">
      <string>Server's ID</string>
     </property>
    </widget>
    <widget class="QLabel" name="portLabel">
     <property name="geometry">
      <rect>
       <x>40</x>
<<<<<<< HEAD
       <y>110</y>
=======
       <y>160</y>
>>>>>>> 0bcd9ef0c187252ff7022416289f373dd0043275
       <width>251</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: italic 600 10pt &quot;UTM Avo&quot;;
border: none;</string>
     </property>
     <property name="text">
      <string>Server's Port</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="portInput">
     <property name="geometry">
      <rect>
       <x>40</x>
<<<<<<< HEAD
       <y>140</y>
=======
       <y>200</y>
>>>>>>> 0bcd9ef0c187252ff7022416289f373dd0043275
       <width>251</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #ffffff;
border: 1px #000000;
border-radius: 5px;
<<<<<<< HEAD
padding: 10px;</string>
=======
padding-left: 10px;</string>
>>>>>>> 0bcd9ef0c187252ff7022416289f373dd0043275
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>40</x>
<<<<<<< HEAD
       <y>240</y>
=======
       <y>310</y>
>>>>>>> 0bcd9ef0c187252ff7022416289f373dd0043275
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 600 10pt &quot;UTM Avo&quot;;
border: 1px solid rgb(7, 7, 248);
background-color: rgb(0, 132, 255);
border-radius: 5px;
color: #ffffff;</string>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>180</x>
<<<<<<< HEAD
       <y>240</y>
=======
       <y>310</y>
>>>>>>> 0bcd9ef0c187252ff7022416289f373dd0043275
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 600 10pt &quot;UTM Avo&quot;;
border: 1px solid rgb(194, 231, 255);
background-color: #ffffff;
border-radius: 5px;
</string>
     </property>
     <property name="text">
      <string>Exit</string>
     </property>
    </widget>
    <widget class="QLabel" name="failLabel">
     <property name="geometry">
      <rect>
<<<<<<< HEAD
       <x>60</x>
       <y>190</y>
       <width>361</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: italic 600 9pt &quot;UTM Avo&quot;;
border: none;
text-align: justify;
color: rgb(170, 0, 0);
</string>
=======
       <x>40</x>
       <y>260</y>
       <width>321</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: bold 600 9pt &quot;UTM Avo&quot;;
color: rgb(170, 0, 0);
border: none;
text-align: justify;</string>
>>>>>>> 0bcd9ef0c187252ff7022416289f373dd0043275
     </property>
     <property name="text">
      <string>Connect unsuccessfully. Try again!</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
<<<<<<< HEAD
     <width>333</width>
=======
     <width>334</width>
>>>>>>> 0bcd9ef0c187252ff7022416289f373dd0043275
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
